### PR TITLE
Pin `torch` in various examples to avoid cuda version issues.

### PR DESCRIPTION
**Pull request by concretevitamin**
Tested:
- Ran on both AWS and GCP.

### CODE REVIEW

File: docs/source/getting-started/tutorial.rst [hunk 1]
```diff
@@ -31,17 +31,16 @@ and the commands to run:
...
   setup: |
     set -e  # Exit if any command failed.
-    conda install pytorch==1.12.1 cudatoolkit=11.3 -c pytorch -y
     git clone https://github.com/huggingface/transformers/ || true
     cd transformers
-    pip3 install .
+    pip install .
     cd examples/pytorch/text-classification
-    pip3 install -r requirements.txt
+    pip install -r requirements.txt torch==1.12.1+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
```
Comment (Michaelvll) [inline docs/source/getting-started/tutorial.rst:38]: Question: Wondering if it works having `conda install` after the `pip install -r requirements.txt`, and why we fall back to the `pip` version

**Review by Michaelvll — APPROVED** [review]
Thanks for the fix @concretevitamin!

**Comment by Michaelvll** [comment]
Can we add the fix to https://github.com/skypilot-org/skypilot/blob/fd6c335d4b2192abe054528513167ef1fedeab73/docs/source/examples/spot-jobs.rst#L57-L58

Also, the yaml file
https://github.com/skypilot-org/skypilot/blob/fd6c335d4b2192abe054528513167ef1fedeab73/examples/spot/bert_qa.yaml#L23-L24

File: docs/source/getting-started/tutorial.rst (hunk 1 above)
Comment (concretevitamin) [reply docs/source/getting-started/tutorial.rst:38]: Tried that out: it worked on GCP but not on AWS.

**Comment by concretevitamin** [comment]
Good catch, thanks!

**Pull request merged**